Libplanet.Tests/Store/StoreTracker.cs

```diff
@@ -118,10 +118,13 @@ public IEnumerable<HashDigest<SHA256>> IterateBlockHashes()
             return _store.IterateBlockHashes();
         }
 
-        public IEnumerable<HashDigest<SHA256>> IterateIndex(string @namespace)
+        public IEnumerable<HashDigest<SHA256>> IterateIndex(
+            string @namespace,
+            int offset,
+            int? limit)
         {
             _logs.Add((nameof(IterateIndex), @namespace, null));
```

Comment:
Shouldn't this be changed either?  Although slots are not enough…

```suggestion
             _logs.Add((nameof(IterateIndex), @namespace, (offset, limit)));
```